keps/2724-topology-aware-scheduling/README.md

```diff
@@ -682,8 +693,8 @@ The new validations which are for MVP, but likely will be relaxed in the future:
 
 #### Beta
 
-- support for all other integrations
-- support multi-level hierarchy
+- support rank-aware packing of pods, so that pods with consecutive ranks are
```

Comment:
Could we move this to Alpha? IIRC, this feature will be released in early Dec or at the end of Node, right?

Reply:
yeah, the plan is early Dec, maybe I move it to "Second Alpha"?

Reply:
OTOH, this is graduation "criteria", rather than strict delivery plan. So, I interpret it - do it before beta. If we do it in second alpha it still meets the "criteria" for beta, so might be fine this way. WDYT?

Reply:
> OTOH, this is graduation "criteria", rather than strict delivery plan. So, I interpret it - do it before beta. If we do it in second alpha it still meets the "criteria" for beta, so might be fine this way. WDYT?

Oh, I see. In that case, I'm ok with the current criteria. 
We can prepare the second Alpha, but it seems like overkill since the period between the first and second alpha is just one month.

/lgtm
/approve